src/sentry/integrations/vsts/issues.py

```diff
@@ -38,7 +38,12 @@ def get_project_choices(self, group, **kwargs):
         project_choices = [(project["id"], project["name"]) for project in projects]
 
         params = kwargs.get("params", {})
-        defaults = self.get_project_defaults(group.project_id)
+        if group:
+            default_project_id = group.project.id
```

Comment:
@ceorourke Nit: you should use `group.project_id` to avoid having to fetch the project